README.rst

```diff
@@ -14,7 +14,7 @@ Policies and Conventions for Organizing Gen3 Data Repositories
 DMTN-167
 ========
 
-While the Gen3 Butler provides some intrinsic structure to its data repositories, considerably more is left to convention (often encoded in higher-level packages, like obs_base).  This document will be - at least at first - a proposal from Science Pipelines leadership on how to organize data repositories in detail, focusing on collection naming conventions, filesystem locations, and developer workflows.  The immediate focus will be the environment at NCSA, but it is hoped that much of this will hold for the IDF and USDF as well.
+While the Gen3 Butler provides some intrinsic structure to its data repositories, considerably more is left to convention (often encoded in higher-level packages, like obs_base).  This document will be - at least at first - a proposal for how to organize data repositories in detail, focusing on collection naming conventions, filesystem locations, and developer workflows.  The immediate focus will be the environment at NCSA, but it is hoped that much of this will hold for the IDF and USDF as well.
```

Comment:
:+1:; I'll add this to the abstract:

> A core assumption is that there will be _one_ large shared data repository for all instruments at NCSA (and each other major facility), for "friendly" use by the DM team.
Smaller custom data repositories may also exist (especially for CI), but we hope to ensure that nearly all development work can be performed without the need for developers to ever create their own personal repositories.
One or more large shared repositories for science users are also expected to exist, but are explicitly beyond the scope of this proposal.

(I think that may also help address your earlier comment about starting with terminology instead of scope.)